Program.cs

```diff
@@ -8,6 +8,7 @@
 using Microsoft.AspNetCore.Http;
 using System.Data.SqlTypes;
 using System.Xml;
+//hello

```

Comment:
what do you think about

```suggestion
//goodbye
```